.editorconfig

```diff
@@ -1,7 +1,12 @@
 root = true
+
 [*]
+charset = utf-8
 end_of_line = lf
 insert_final_newline = true
 indent_style = space
 indent_size = 4
 trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
we don't rely on that feature. i would not allow for trailing spaces anywhere except fixture files